### DISCUSSION TITLE
`publish: hidden` shows no effect

### DISCUSSION THREAD

**Discussion by vamonk** [mkdocs-publisher/mkdocs-publisher#100] [Q&A] [answered]
The other parameters such as `title` and `description` are recognized, so the plugin works. Only `publish: hidden` has no effect. It is always published. 
Is this a bug or am I doing something wrong? 

My `mkdocs.yml` is very simple: 

```
site_name: D-MASH Dokumentation

plugins:
    - pub-obsidian
    - pub-meta
```

**Comment by mkusz** [comment, marked as answer]
Hidden has no effect when you do `mkdocs serve`. Also `publish: false` doesn't work while doing it. It's because serve is used always during writing pages, so you can have live preview.
When you do `mkdocs build` and see build page, all hidden ones should be hidden and draft one, should not be accessible.
I will consider adding a switch to turn it off in `pub-meta` config.

**Reply by vamonk** [reply]
Ah ok, thank you!

**Comment by mkusz** [comment]
I will add information about it to the documentation and then close this discussion.
